src/preferences/configobject.cpp

```diff
@@ -270,6 +293,24 @@ void ConfigObject<ConfigValue>::setValue(
     set(key, ConfigValue(QString::number(value)));
 }
 
+template<>
```

Comment:
what does this double template<> idiom mean?

Reply:
I don't know, I'm no C++ expert. I just copied that from all the other functions in the file.

Reply:
Template specialization

Reply:
It is a double specialization, for the class and the function. 
ConfigObject<ValueType>::setValue<ResultType> 
The angle brackest after setValue is omitted because it is defined explicitly by he parameter types.